ha/core/controllers/pcs/node_controller.py

```diff
@@ -242,12 +304,45 @@ def start(self, nodeid: str) -> dict:
             Log.error(f"{nodeid} status is {_node_status}, node may not be started.")
             raise ClusterManagerError(f"Failed to start {nodeid} as found unhandled status {_node_status}")
 
-class PcsHWNodeController(PcsVMNodeController):
+    @controller_error_handler
+    def stop(self, node_id: str, timeout: int= -1, **op_kwargs) -> dict:
```

Comment:
node_id

use either nodeid or node_id for all interface

Reply:
As per ha_python_api used node_id here
def stop(self, node_id: str, **op_kwargs) -> dict